### DISCUSSION TITLE
Move application from web to electron

### DISCUSSION THREAD

**Discussion by Manku27** [Manku27/hotel-admin#12] [Ideas]
For the initial register feature, web was good enough.
But new features requirements are as follows :

1. We are moving to integrate OCR, file system operations will come in handy.
2. Lack of internet cannot interfere with operations. We will sync with our servers but local databases will be required.
3. Also, our customer base thinks software means executables, web seems unprofessional to them.

**Comment by cheenashe** [comment]
Great choice! Electron will handle OCR and file system tasks well, plus the offline capabilities and local databases are spot on for your needs. It’s also a good way to give your customers the executable experience they expect. 

If you need any tips or have questions about the transition, feel free to reach out. Happy to help with [web application development](https://www.sygitech.com/web-application-development.html) or any other aspects of this shift!

**Reply by Manku27** [reply]
Thanks for validating my idea.

I would love to get all the help I can, but I am not sure if you are talking in an individual capacity or on behalf of your organization.
I can't pay for any help.
